lib_lore/example_use/pack.mcmeta

```diff
@@ -13,8 +13,8 @@
     "credits": {
         "Creator": [
             [
-                "BluePsychoRanger",
-                "https://twitter.com/BluPsychoRanger"
+                "BPR",
+                "https://bsky.app/profile/bpr02.com"
             ]
         ]
```

Comment:
Should our libraries also use the shorthand system and `contributors.json` or is that bad as it doesn't show the link when someone copies the raw files?